website/content/en/docs/kubebuilder/migration/project_migration_guide.md

```diff
@@ -0,0 +1,167 @@
+# Overview
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
+- [git][git_tool].
+- [go][go_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [kustomize][kustomize_tool] v3.1.0+.
+- Access to a Kubernetes v1.11.3+ cluster. 
+
+**Note**
+It is recommended that you have your project upgraded to the latest SDK release version before following the steps of this guide to migrate to Kubebuilder layout.
+
+## Install the Operator SDK CLI
+
+Follow the steps in the [installation guide][install_guide] to learn how to install the Operator SDK CLI tool.
+
+## Create a new project
+
+Use `operator-sdk init` command to create a new project:
+
+For example, to create a Memcached operator project:
+
+```sh
+$ mkdir $HOME/projects/memcached-operator
+$ cd $HOME/projects/memcached-operator
+$ operator-sdk init --domain example.com --repo github.com/example-inc/memcached-operator
+$ cd memcached-operator
+```
+
+Make sure to activate the go module support by running `$ export GO111MODULE=on`.
+
+**Note**
+If you intend to have multiple groups in your project, then add the line `multigroup: true` in the `PROJECT` file. The `PROJECT` file for the above example would look like:
+
+```
+domain: example.com
+repo: github.com/example-inc/memcached-operator
+multigroup: true
+version: 2
+...
+```
+For multi-group project, the Kind API's files are created under `apis/<group>/<version>` and the controllers would be created under `controllers/<group>`.
+
+## Create an API
+
+Create a new API and a new kind (CRD) for the operator.
+
+In case of memcached operator, a new API named `cache/v1aplha1` (group/version) and a new kind `Memcached` is created using the following command.
```

Comment:
group/version and kind altogether define an API. Kind isn't really it's own thing.
```suggestion
In case of memcached operator, a new API for  kind `Memcached` and group/version `cache/v1aplha1` is created using the following command.
```